buildSrc/src/main/java/org/opensearch/gradle/zipplugin/ZipPublish.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.gradle.zipplugin;
+
+import java.util.*;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.publish.PublishingExtension;
+import org.gradle.api.publish.maven.MavenPublication;
+import org.gradle.api.publish.maven.plugins.MavenPublishPlugin;
+import java.nio.file.Path;
+import org.gradle.api.Task;
+
+public class ZipPublish implements Plugin<Project> {
+    private Project project;
+
+    public final static String EXTENSION_NAME = "zipmavensettings";
+    public final static String PUBLICATION_NAME = "mavenzip";
+    public final static String STAGING_REPO = "zipstaging";
+    public final static String MAVEN_ZIP_PUBLISH_TASK = "publish"
+        + ZipPublishUtil.capitalize(PUBLICATION_NAME)
+        + "PublicationTo"
+        + ZipPublishUtil.capitalize(STAGING_REPO)
+        + "Repository";
+    public final static String MAVEN_ZIP_PUBLISH_POM_TASK = "generatePomFileFor"
+        + ZipPublishUtil.capitalize(PUBLICATION_NAME)
+        + "Publication";
+    public final static String LOCALMAVEN = "publishToMavenLocal";
+    public final static String LOCAL_STAGING_REPO_PATH = "/build/local-staging-repo";
+    public static String BUILD_DISTRIBUTIONS_LOCATION = "/build/distributions/";
+
+    private void configMaven() {
+        final Path buildDirectory = this.project.getRootDir().toPath();
+        this.project.getPluginManager().apply(MavenPublishPlugin.class);
+        this.project.getExtensions().configure(PublishingExtension.class, publishing -> {
+            publishing.repositories(repositories -> {
+                repositories.maven(maven -> {
+                    maven.setName(STAGING_REPO);
+                    maven.setUrl(buildDirectory.toString() + LOCAL_STAGING_REPO_PATH);
+                });
+            });
+            System.out.println("Starting " + MAVEN_ZIP_PUBLISH_TASK + " task");
```

Comment:
Remove `System.out.println`.

Reply:
Will be removing in my next commit.

Reply:
Just removed. it and also cleaned up the code   
 
 ```
public final static String MAVEN_ZIP_PUBLISH_TASK = "publish"
        + ZipPublishUtil.capitalize(PUBLICATION_NAME)
        + "PublicationTo"
        + ZipPublishUtil.capitalize(STAGING_REPO)
        + "Repository";

```